tests/RestControllerTest.php

```diff
@@ -340,4 +340,33 @@ public function testLabelNoConcept() {
       $this->assertEquals($expected, $out);
   }
 
+  /**
+   * @covers RestController::newConcepts
+   */
+  public function testNewConcepts() {
```

Comment:
The test name is confusing. It's called `testNewConcepts` but it actually calls `modifiedConcepts`. Also the `@covers` annotation above is wrong. In any case, there should be tests like this both for `newConcepts` and `modifiedConcepts`.